src/components/Timeline/TimelineBrowser.js

```diff
@@ -314,20 +299,27 @@ const TimelineBrowser = ({ timelineIntervals }) => {
             maxCount1SD={maxCount1SD}
             maxCount2SD={maxCount2SD}
           />
-          <RecordAndIntervalCount
+          <Metrics
             availableWidth={availableWidth}
-            intervalCount={intervalCount}
-            intervalsWithRecordsCount={intervals.length}
-            recordCount={recordCount}
+            intervalLength={intervalLength}
           />
           <Rect
             x="0"
             y="0"
             width={availableWidth}
-            height={BAR_HEIGHT + 14}
+            height={BAR_HEIGHT}
+            strokeDasharray="2 2"
             // stroke="#f008" // debug position
           />
         </G>
+        <Rect
+          x="0"
+          y="0"
+          width={screenWidth}
+          height={CHART_HEIGHT}
+          strokeDasharray="2 2"
+          // stroke="#00f8" // debug position
```

Comment:
keeping these for debugging; when enabled, the boundaries seem unexpected.

...this relates to:

https://trello.com/c/naiYro4M/199-responsive-timeline-height-for-any-device-size